CMakeLists.txt

```diff
@@ -445,7 +449,46 @@ if(AVIF_CODEC_AOM)
     message(STATUS "libavif: Codec enabled: aom (${AVIF_CODEC_AOM_ENCODE_DECODE_CONFIG})")
 endif()
 
-if(NOT AVIF_CODEC_AOM AND NOT AVIF_CODEC_DAV1D AND NOT AVIF_CODEC_LIBGAV1)
+if(AVIF_CODEC_AVM)
+    message(WARNING "libavif: AV2 support with avm is experimental. Only use for testing.")
+
+    # The avm repository is a fork of aom and inherited a lot of folders, files and build artifacts named the same way.
+    # Having both dependencies at the same time generates conflicts in includes, binary lookups etc.
```

Comment:
This is problematic. If it is not feasible to implement AV2 as a codec algorithm in the same aom_codec_ interface, the AV2 team should rename the public headers, libraries, and global functions and variables in AVM.

Reply:
Thank you. I just added a comment to that bug.